include/picongpu/CMakeLists.txt

```diff
@@ -247,6 +249,19 @@ if(ADIOS_FOUND)
     set(LIBS ${LIBS} ${ADIOS_LIBRARIES})
 endif(ADIOS_FOUND)
 
+################################################################################
+# openPMD
+################################################################################
+
+# find openPMD installation
+find_package(openPMD 0.8.0)
+
+if(openPMD_FOUND)
+    add_definitions(-DENABLE_OPENPMD=1)
+    include_directories(SYSTEM ${openPMD_INCLUDE_DIRS})
```

Comment:
this line is not needed, because the linked `openPMD::openPMD` target carries its includes as property.
```suggestion
```